internal/cmdrender/executor.go

```diff
@@ -75,7 +75,8 @@ func (e *Executor) Execute(ctx context.Context) error {
 		return err
 	}
 
-	pr.PkgPrintf(root.pkg.UniquePath, "rendered successfully\n")
+	pr.Printf("Successfully executed %d function(s) in %d package(s).\n", hctx.executedFunctionCnt, len(hctx.pkgs))
```

Comment:
We can add expected StdOut for one of the happy path tests to cover this one ?

Reply:
added the test case